includes/rules/color_contrast_failure.php

```diff
@@ -346,13 +347,9 @@ function edac_test_color_diff( $color1, $color2 ) {
 		0.7152 * pow( $color2['g'] / 255, 2.2 ) +
 		0.0722 * pow( $color2['b'] / 255, 2.2 );
 
-	if ( $l1 > $l2 ) {
-		$dif = ( ( $l1 + 0.05 ) / ( $l2 + 0.05 ) ) + 0.05;
-	} else {
-		$dif = ( ( $l2 + 0.05 ) / ( $l1 + 0.05 ) ) + 0.05;
-	}
-
-	return $dif;
+	return ( $l1 > $l2 )
+		? ( ( $l1 + 0.05 ) / ( $l2 + 0.05 ) ) + 0.05
+		: ( ( $l2 + 0.05 ) / ( $l1 + 0.05 ) ) + 0.05;
```

Comment:
Technically this is the same as using `else`, but it makes the code easier to read and it's easier to see at a glance what the difference between the 2 actually is 👍